client/my-sites/checkout/checkout-thank-you/test/index.js

```diff
@@ -30,14 +30,16 @@ jest.mock( 'calypso/lib/analytics/page-view-tracker', () => () => 'page-view-tra
 jest.mock( '../header', () =>
 	jest.fn( ( { children } ) => <div data-testid="checkout-thank-you-header">{ children }</div> )
 );
+// eslint-disable-next-line react/display-name
 jest.mock( 'calypso/components/happiness-support', () => () => (
 	<div data-testid="happiness-support" />
 ) );
-jest.mock( 'calypso/components/wordpress-logo', () => () => <div data-testid="wordpress-logo" /> );
 jest.mock( '../transfer-pending/', () => () => 'transfer-pending' );
+// eslint-disable-next-line react/display-name
 jest.mock( '../redesign-v2/pages/plan-only', () => () => (
 	<div data-testid="component--plan-only-thank-you" />
 ) );
+// eslint-disable-next-line react/display-name
```

Comment:
Fixes the [ratcheted lint errors](https://teamcity.a8c.com/viewLog.html?buildId=14831321&tab=Inspection&buildTypeId=calypso_calypso_WebApp_Check_Code_Style_Branch). Suppressing the error because it doesn't matter here.